woocommerce/woocommerce-android
# Save order filters into shared preferences

Work in progress for: #4934

### Description
Save selected order filters into shared preferences. 

### Testing instructions
- Open order list
- Select some filters 
- Kill the app
- Open the app again and check that filter selection is kept

IMPORTANT. Since this feature is still WIP and we want to merge things progressively (under feature flag protection) you'll see that when opening order list again after killing the app, while the filters are properly restored, the order list is not refreshed accordingly with the selected filters. This is because we have to keep to keep the "old code" functional with the 2 tabs for order list. Until I remove that completely the feature won't be 100% operational. 

Also, filters are kept between site changes so you can also test changing site and when coming back to the previously selected site the filters will be there. 

### Demo

https://user-images.githubusercontent.com/2663464/140339168-1feb8a73-3de5-4d38-8fb7-be63fc117e8c.mp4

https://user-images.githubusercontent.com/2663464/140342766-92ced5ff-3c07-45ed-8189-cb0d085603ee.mp4
